extensions/applicationinsights-react-native/src/ReactNativePlugin.ts

```diff
@@ -7,11 +7,17 @@ import {
     ITelemetryPlugin,
     ITelemetryItem,
     IPlugin,
-    IConfiguration,
-    IAppInsightsCore
+    IAppInsightsCore, 
+    ICustomProperties,
+    CoreUtils,
+    IDiagnosticLogger,
+    DiagnosticLogger,
+    LoggingSeverity,
+    _InternalMessageId
 } from '@microsoft/applicationinsights-core-js';
-import { ConfigurationManager, IDevice } from '@microsoft/applicationinsights-common';
+import { ConfigurationManager, IDevice, IMetricTelemetry, IAppInsights, IExceptionTelemetry } from '@microsoft/applicationinsights-common';
 import DeviceInfo from 'react-native-device-info';
+import { setJSExceptionHandler, setNativeExceptionHandler } from 'react-native-exception-handler';
```

Comment:
This is a breaking change for anyone that upgrades this plugin and does not have/want this package. Maybe we can lazy load this and the device-info package and only enable them if they are available?